fpcollect/Transport_Factory.cpp

```diff
@@ -1,8 +1,7 @@
 #include "Transport_Factory.h"
 #include <spipc/Socket_Transport.h>
 #include <utils.h>
-//#include "Mongo_Storage.h"
-#include "Mongo_stub.h"
```

Comment:
I guess Sabina just did not commit stubs.. I would prefer that stub is still available and could be used when needed, could you please import stub files - could be useful.